warden/src/assignment_computation.rs

```diff
@@ -84,15 +87,28 @@ pub struct AssignmentComputationImpl {
     ready_range_servers: Mutex<HashSet<HostInfoWrapper>>,
     unassigned_base_ranges: Mutex<Vec<RangeInfo>>,
     assignment_update_sender: Sender<i64>,
+    session: Session,
 }
 
+static INSERT_INTO_RANGE_LEASE_QUERY: &str = r#"
+  INSERT INTO chardonnay.range_leases(range_id, key_lower_bound_inclusive, key_upper_bound_exclusive)
+    VALUES (?, ?, ?)
+    IF NOT EXISTS
+"#;
+
```

Comment:
Can we move all the cassandra stuff into the persistence module instead of having it in the assignment_computation directly?

Reply:
Good idea. I moved it.